source/intro_to/explore_software.rst

```diff
@@ -0,0 +1,229 @@
+
+:bash:`N4BiasFieldCorrection` To perform bias correction algorithm on T1w, which evaluate the spatial change in signal and ensure its uniformity.
+
+:bash:`antsAtroposN4.sh` To segment WM-GM-CSF and to compute cortical thickness. This is a less common usage in the lab.
+
+
+Installation
+^^^^^^^^^^^^^^^^^^^^^
+
+Visit this website for download and install instructions `(here) <https://github.com/ANTsX/ANTs/wiki/Compiling-ANTs-on-Linux-and-Mac-OS>`__. ANTs use a pretty complex algorithm, so the command line is sometimes hard to understand at first, `this <https://github.com/ANTsX/ANTs/wiki/Anatomy-of-an-antsRegistration-call>`__. should help.
+
+.. code-block:: bash
+
+    export ANTSPATH="/PATH/TO/ANTS/ants/build/install/bin"
+    export PATH=${ANTSPATH}:${PATH}
+
+Once installed, you should be able to type :bash:`antsRegistration` to see the help display.
+
+MRtrix
+-----------------------
+MRtrix is an open-source software package for diffusion MRI analysis and Tractography. It provides a suite of tools for processing, analyzing, and visualizing diffusion MRI data and streamlines. It is widely used in research and clinical applications.
+Visit this `website <https://www.mrtrix.org/>`__ for download and install instructions as well as a general overview of tools available in `documentation <https://mrtrix.readthedocs.io/en/latest/>`__
+
+For a quick overview of commands you can look at their Getting Started `(here) <https://mrtrix.readthedocs.io/en/latest/getting_started/beginner_dwi_tutorial.html>`__ section. This `webpage <https://osf.io/fkyht/>`__  hosts an extensive tutorial of MRtrix3 from raw data to tractography (and more). We recommend trying this tutorial and finding analogous functions in Scilpy to reach the same end goals.
+
+MRtrix is another set of tools to analyse diffusion data, much like Dipy and Scilpy. One main difference is that they use C++ whereas we use python.
+
+Useful Commands
+^^^^^^^^^^^^^^^^^^^^^
+
+:bash:`tckgen`: a tool for tractography, generating streamlines based on the diffusion MRI data.
+
+:bash:`dwidenoise`: a tool for removing noise from the diffusion MRI data.
+
+:bash:`mrconvert`: a tool for converting between different file formats of MRI data. Changing strides and datatype. This is a useful tool to convert DICOM to NIFTI when other tools.
+
+:bash:`mrinfo`: a tool for displaying information about an MRI dataset, such as image dimensions, voxel sizes, and data types.
+
+:bash:`mrview`: a viewer for visualizing image and tractography file, fast and intuitive for fast sanity check as well as advanced rendering.
+
+MRtrix pipeline is very similar in spirit to the Scilpy/Dipy and Tractoflow pipeline we implement at the SCIL. 
+Both tools were created around the same time. Both are open source. Both are using a script approach.
```

Comment:
This and the following paragraphs are a bit redundant after  line 146. I would merge.